src/main/resources/db/migration/V1__create_client_table.sql

```diff
@@ -1,11 +1,11 @@
 CREATE TABLE `client` (
-    clientId int AUTO_INCREMENT NOT NULL,
+    clientId int AUTO_INCREMENT NOT NULL PRIMARY KEY,
     name nvarchar(50) NOT NULL,
-    addressLine1 nvarchar(50) NOT NULL,
-    addressLine2 nvarchar(50),
-    town nvarchar(50),
-    city nvarchar(50),
-    postcode nvarchar(10),
-    country char(2) NOT NULL
+    addressLine1 varchar(50) NOT NULL,
+    addressLine2 varchar(50),
+    town varchar(50),
+    city varchar(50),
+    postcode varchar(10),
+    country char(2) NOT NULL,
     phone varchar(15)
```

Comment:
if we are doing nvarchar for the rest why not here? also can this be null? do we have a form of contact?

Reply:
can what be null? is phone number not a form of contact?